### ISSUE TITLE
How to handle iced runtime?

### ISSUE THREAD

**Issue by 0xorial** [patrickxb/gulp-iced#6]
How do I add iced runtime to my page? I assume that 'inline' option makes it inlined with each file.. Is there a better way if there are multiple files in project?

**Comment by doublerebel** [comment]
@0xorial:  yes, better than `'inline'` is `'window'`.  As described in the [Iced docs](http://maxtaco.github.io/coffee-script/) under Usage:

> If window is specified, make inline a runtime and assigned it to window.iced, so that other fragments later in the file don't need it.
